supervision/detection/annotate.py

```diff
@@ -147,6 +133,128 @@ def annotate(
         return scene
 
 
+class CircleAnnotator(BaseAnnotator):
```

Comment:
I think that `CircleAnnotator` should only draw circles.

In the next PR, we should add `TextAnnotator` responsible for drawing labels with background. `TextAnnotator` could be used in combination with `CircleAnnotator`, `TriangleAnnotator`, `EllipseAnnotator`, and so on.

@onuralpszr please drop the whole text drawing logic, and keep only the circles drawing.

Reply:
@SkalskiP  I will do changes but can we merge other PR first then I can do changes after rebase and give you clean results ? or I can add this codes to over there with your changes and also keep clean ?

Reply:
By `other PR` you mean #170?